docs/2022-07-18-JSON-RPC-Server-API.md

```diff
@@ -192,10 +199,65 @@ If `"reason": "cut-point-rule"`, the `next-states` field contains the next state
         "predicate": {"format":"KORE", "version":1, "term":{}},
         "substitution": {"format":"KORE", "version":1, "term":{}},
       }
-    ]
+    ],
+    "logs": []
+  }
+}
+```
+
+The `logs` field in the correct responses (if not empty) will contain objects of the following structure:
```

Comment:
Can we make mention of any guarantee on teh order we can assume from the log entries?

Reply:
can you think of use-cases for this? I would like to keep it a flat list for now unless we find a compelling reason to build in such hierarchy

Reply:
It's just that if these ordering assumptions are part of the contract, then it's encoded in the schema, and not just in documentation.

But it's not critical, the client can also transform the data to this format if needed.